templates/bundles/TwigBundle/Exception/error.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends '@W3CWebsiteTemplates/base.html.twig' %}
+
+{% block main %}
+    <h1>{% block title %}{{ page.title|default('W3C - Error: {{ status_code }} - {{ status_text }}') }}{% endblock %}</h1>
```

Comment:
Write "W3C" at the end of the title string to be coherent with other titles.

 ```suggestion
     <h1>{% block title %}{{ page.title|default('Error: {{ status_code }} - {{ status_text }}') }} | W3C{% endblock title %}</h1>
```

Reply:
And that sentence also need no be localized.

Reply:
should we only translate 'Error:' or also plan to translate the `status_text`?

Reply:
`status_text` itself probably do not need to be localized, but the whole sentence should be and not just the `Error` word, as depending on the language that word might be at the start or the end of the sentence.

Reply:
addressed in https://github.com/w3c/w3c-website-templates-bundle/pull/40/commits/776d66e897902c8c85388521c3a2561fa634b319

Reply:
Addressed by @deniak in https://github.com/w3c/w3c-website-templates-bundle/pull/40/commits/776d66e897902c8c85388521c3a2561fa634b319